packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart

```diff
@@ -20,6 +21,113 @@ import 'scroll_position.dart';
 import 'scroll_position_with_single_context.dart';
 import 'scrollable.dart';
 
+/// Class that supplies children for the [ListWheelScrollView].
+///
+/// Instead of providing a concrete list of children to [ListWheelScrollView],
+/// this class allows several methods to lazily provide children, which helps
+/// when there is an infinite amount of children.
+abstract class ChildDelegate {
```

Comment:
Either make this name reflect that it's specifically about list wheel scroll views or don't bring in anything list wheel scroll view related in here and put it in a generic class.